mission_control/navigator_launch/launch/bag.launch

```diff
@@ -1,10 +1,16 @@
 <launch>
```

Comment:
Some more things we might want in here to test with bags real easy

- optionally launch perception.launch
- optionally launch shooter